style.css

```diff
@@ -671,14 +798,131 @@ div {
     margin-bottom: 127px;
   }
 
-  .error {
+  /* Popup Window */
+
+  .popup-cards-container {
     display: none;
-    color: rgb(190, 54, 54);
-    width: 37%;
-    margin: 0 auto;
-    position: relative;
-    bottom: 180px;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    padding: 5px;
+    z-index: 99999;
+    background-color: #c1c7d0;
+  }
+
+  .popup-cards {
+    border-radius: 16px;
+    border: 1px solid var(--neutrals-n-30-l, #ebecf0);
+    background: var(--white, #fff);
+    box-shadow: 0 48px 48px 0 rgba(37, 47, 137, 0.08);
+    margin: 0 24px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 12px;
+  }
```

Comment:
- [ ] Good job implementing the changes and improving the UI of the project :clap:  however, there are still some adjustments to be made. Currently, your popup does not align with the design.  Kindly make some adjustments to your styling to implement these changes. Please check the image attached below for comparison and better understanding. :slightly_smiling_face:  

Below is an illustration of the difference: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

![Student Popup](https://github.com/78fatemanazari/Portfolio/assets/43528594/c1eff953-de40-4d16-bc75-4d6d75262ca8)

</td>
      <td>

![Figma Desktop](https://github.com/78fatemanazari/Portfolio/assets/43528594/5fe47f01-9414-4f15-8609-773caa8e89f1)


</td>
    </tr>
  </table> 


<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

![Student Mobile Popup](https://github.com/78fatemanazari/Portfolio/assets/43528594/589ea67c-2135-4c64-9270-5b3c692ec66a)

</td>
      <td>

![Figma mobile popup](https://github.com/78fatemanazari/Portfolio/assets/43528594/3c14c4b4-cd87-438d-a5e7-6d14791efbf3)

</td>
    </tr>
  </table>